movieland_persistence_jdbc/src/main/java/com/solovey/movieland/dao/jdbc/JdbcReviewDao.java

```diff
@@ -36,13 +40,26 @@
     }
 
     @Override
-    public void saveReviewToDb(Review review) {
+    public Review saveReviewToDb(Review review) {
         log.info("Start inserting review {} into DB", review.toString());
         long startTime = System.currentTimeMillis();
-        jdbcTemplate.update(insertReviewSql, review.getMovieId(),
-                review.getUser().getId(),review.getText());
+        KeyHolder keyHolder = new GeneratedKeyHolder();
+
+
+        jdbcTemplate.update((connection)->{
```

Comment:
you can try to use NamedParameterJdbcTemplate

Reply:
done